optaplanner-docs/src/main/asciidoc/MoveAndNeighborhoodSelection/MoveAndNeighborhoodSelection-chapter.adoc

```diff
@@ -259,13 +264,14 @@ Advanced configuration:
 [source,xml,options="nowrap"]
 ----
     <pillarChangeMoveSelector>
+      <subPillarType>SEQUENCE</subPillarType>
+      <subPillarSequenceComparatorClass>...</subPillarSequenceComparatorClass>
```

Comment:
Provide an actual value in here, so users see an example how it could look like.
Best is to copy paste something from an example, such as nurse rostering, but then change Lecture with ShiftAssignment.

Reply:
See also docs of threadFactoryClass, customPhaseCommandClass, etc

Reply:
@triceo I don't see this changed? You marked it as resolved - did this slip through or do you reject my comment (which is fine if you motivate it).